strapi/strapi
# [DEITS] Entities transfer + fixes

### What does it do?

- Fix entities export from the strapi local source provider
- Use a shared set of entity & component utils for strapi providers (bypass the entity service but handle cascade components)
- Update tests
- Modify the restore file layout to separate the different stages
- Implement the entities restore (update deletion params & mechanisms + create)
- Take into account the transforms steps that can be configured for the engine & update the import + export commands to use them

### Why is it needed?

- entities export is broken
- need to implement the entities' import
- factorize utilities & simplify code
- fix tests 

### How to test it?

- yarn strapi export --no-encrypt -f backup
- yarn strapi import -f backup.tar.gz
- You should see both entities & configuration being imported in your database

Note: The content manager will appear broken after the entities are imported. It might be linked to the fact that the links are still not imported
